README.md

```diff
@@ -309,6 +309,29 @@ global:
         keep: true # keep the attribute with the original key
 ```
 
+### <a name="probabilistic-trace-sampling"></a>Probabilistic Head-based Trace Sampling
+
+In some scenarios it may be desirable to perform probabilistic head-based trace sampling on the collector.
+This can be done using by specifying `probabilistic` policy secion under the `sampling` section of the collector configuration file.
+
+```yaml
+sampling:
+  # mode indicates if the sampling is head or tail based. For probabilistic the mode is head-based.
+  mode: head
+  policies:
+    # section below defines a probabilistic trace sampler based on hashing the trace ID associated to
+    # each span and sampling the span according to the given spans.
+    probabilistic:
+      configuration:
+        # sampling-percentage is the percentage of sampling to be applied to all spans, unless their service is specified
+        # on sampling-percentage.
+        sampling-percentage: 5
+        # hash-seed allows choosing the seed for the hash function used in the trace sampling. This is important when
+        # multiple layers of collectors are being used with different sampling rates, in such scenarios make sure to
```

Comment:
Does it matter that sampling rate is different? If I have a chain of collectors configured with the same sampling rate and hash seed wouldn't it result in downstream collectors simply parroting upstream ones and sampling 100%? I may be me misunderstanding how the feature works, but it probably means we need better doc since our users will likely misunderstand too.

Reply:
You're right, it doesn't matter. Let me attempt a rewrite of this.